packages/paste-website/stories/PageTemplates.stories.tsx

```diff
@@ -860,3 +880,505 @@ export const SettingsOverviewExample = (): JSX.Element => {
+        <Box display="flex" width="100%" justifyContent="center" paddingX="space50" id={mainContentSkipLinkID}>
+          <Box
+            paddingTop="space130"
+            paddingBottom="space160"
+            display="flex"
+            flexDirection="column"
+            rowGap="space130"
+            width="size70"
+            maxWidth="size80"
+          >
+            <Box display="flex" flexDirection="column">
+              <DetailText>Add Reverse ETL destination</DetailText>
+              <Heading as="h4" variant="heading10" marginBottom="space0">
+                Set up destination
+              </Heading>
+            </Box>
+            <Box>
+              <Form>
+                <FormControl>
+                  <Label htmlFor={input1}>Destination name</Label>
+                  <Input type="text" id={input1} name="input1" />
+                  <HelpText>
+                    Identifies this destination within your organization and typically includes the business unit,
+                    geographic region, or environment. Example: Google Analytics North America Prod.
+                  </HelpText>
+                </FormControl>
+              </Form>
+            </Box>
+            <Box display="flex" justifyContent="space-between">
+              <Button variant="secondary">
+                <ArrowBackIcon decorative />
+                Back
+              </Button>
+              <Box display="flex" columnGap="space40">
+                <Button variant="secondary">Cancel or Close</Button>
+                <Button variant="primary">Create destination</Button>
+              </Box>
+            </Box>
+          </Box>
+        </Box>
+      </SidebarPushContentWrapper>
+    </>
+  );
+};
+DefaultWizardWithContentExample.parameters = {
+  padding: false,
+};
+
+export const DefaultWizardOrderedListExample = (): JSX.Element => {
+  const input1 = useUID();
+  const input2 = useUID();
+  const input3 = useUID();
+  const input4 = useUID();
+  const input5 = useUID();
+  return (
+    <Box display="flex" width="100%" justifyContent="center" paddingX="space10">
+      <Box
+        paddingTop="space130"
+        paddingBottom="space160"
+        display="flex"
+        flexDirection="column"
+        rowGap="space130"
+        width="size70"
+        maxWidth="size80"
+      >
+        <Box display="flex" flexDirection="column">
+          <ProgressSteps>
+            <ProgressStepCurrent as="div">Label</ProgressStepCurrent>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
```

Comment:
todo: Let's make these real with the 4 actual steps & make sure the content doesn't break the layout:

- Select warehouse type
- Connect warehouse
- Set sync schedule
- Set selective sync